### PR TITLE
Add media download functionality to exportActorProfile

### PR DESCRIPTION
**Pull request by 0marSalah**
add media/ folder to exported tarball file 

media files after extracting path => media/{media.id}.{file_extension}

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -18,16 +30,21 @@ export function exportActorProfile({
   blockedAccounts,
   blockedDomains,
   mutedAccounts
-}: ActorProfileOptions): tar.Pack {
-  const pack: Pack = tar.pack() // pack is a stream
+}: ActorProfileOptions & { media?: File[] }): Promise<{
+  addMediaFile: (
```
Comment (dmitrizagidulin) [inline src/index.ts:34]: are you defining this function embedded in the other one? Extract it to be the same level.

Comment (0marSalah) [reply src/index.ts:34]: i have extracted addMediaFile into its own function, which takes pack and manifest as arguments

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Comment by dmitrizagidulin** [comment]
Published on NPM as `v0.1.5`

**Pull request merged**